docs/src/components/App/Header/Header.less

```diff
@@ -140,3 +140,14 @@
     margin-bottom: @row-height;
   }
 }
+
+.highlightWrapper {
+  position: relative;
+}
+
+.highlightedIcon {
+  position: absolute;
+  top: 16px;
+  left: -24px;
+  transform: rotate(-90deg);
```

Comment:
This style is available via a prop in chevronIcon https://github.com/seek-oss/seek-style-guide/blob/master/react/ChevronIcon/ChevronIcon.js#L23